### PR TITLE
Admin newactivity

### PR DESCRIPTION
**Pull request by 12aayub**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,21 +0,0 @@
-# See https://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (orangewolf) [inline .gitignore]: this file is kind of important, where did it go?

File: front-end/src/pages/newActivity.js [hunk 2]
```diff
@@ -31,15 +31,18 @@ class NewActivity extends Component {
   }
 
   handleSubmit(){
-    this.props.dispatch(createNewActivity(this.props.apiUrl, this.state.form))
+    this.props.onSubmit(this.state.form)
+    this.setState({form:{ name: '', description:'', address: '', latitude:'', longitude:''}})
+    document.getElementById("submitMessage").innerText = "Activity Submitted"
+    setTimeout(function(){document.getElementById("submitMessage").innerText = ""}, 2000)
```
Comment (orangewolf) [inline front-end/src/pages/newActivity.js]: why a timeout here?

Comment (kzomkw) [reply front-end/src/pages/newActivity.js]: To remove the "Activity Submitted" text after it displays for 2 seconds.

File: .gitignore (hunk 1 above)
Comment (kzomkw) [reply .gitignore]: re-added!

**Pull request merged**